### ISSUE TITLE
Describe ResolveInfo as per latest conclusions

### ISSUE THREAD

**Issue by dslmeinte** [LionWeb-io/specification#96]
We concluded 2023-03-10 (as part of the discussion on #36) that a reference is always serialized as an object with either an ID or a `resolveInfo` (with a name or something) - at least one of these. In case both are provided the ID takes precedence!

**Comment by enikao** [comment]
@dslmeinte Is this done?

**Comment by dslmeinte** [comment]
I don't think so, since the precedence in case of presence of both is not described here: https://github.com/LIonWeb-org/organization/blob/main/serialization/serialization.adoc
